### DISCUSSION TITLE
求助，绳网id怎么改啊？？

### DISCUSSION THREAD

**Discussion by xiaoqi-1999** [share121/inter-knot#1022] [其他]
如题，如何更改绳网id？

**Comment by lllliufeng** [comment]
头像→Settings→Account→Change username
注意只能是字母数字和连字符

**Comment by isyuah** [comment]
github昵称

**Comment by zhengyang3552** [comment]
进入github → 右上角头像 → Settings → Account → Change username → 点击 Change username 按钮 → 会弹出 Really change your username? 弹窗 → 点击 I understand, let’s change my username 按钮 → 在 Enter a new username 输入框输入你想更改的名称
**注：如果输入中文名则会出现 Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen.（用户名只能包含字母数字字符或单个连字符，并且不能以连字符开头或结尾。）**

**Comment by xiaoqi-1999** [comment]
已解决，谢谢大佬们。
